src/api/print.api.js

```diff
@@ -92,6 +92,23 @@ class GeoAdminCustomizer extends BaseCustomizer {
         symbolizer.pointRadius = adjustWidth(symbolizer.pointRadius, this.printResolution)
         symbolizer.strokeWidth = adjustWidth(symbolizer.strokeWidth, this.printResolution)
         symbolizer.haloRadius = adjustWidth(symbolizer.haloRadius, this.printResolution)
+
+        // we try to adapt the font size and offsets to have roughly the same
+        // scales on print than on the viewer.
+        try {
+            symbolizer.labelYOffset = symbolizer.labelYOffset
+                ? adjustWidth(symbolizer.labelYOffset, this.printResolution)
```

Comment:
might be a better idea to make `adjustWidth` to return `0` if the input is falsy.
I don't really like multiple ternary operator in a row, that makes the code a bit harder to read/understand